static/src/javascripts/lib/geolocation.js

```diff
@@ -49,7 +52,7 @@ const init = (): void => {
             ),
         });
         // Allows application logic to wait until geolocation is set.
-        mediator.emit(geoLocationSetEvent);
+        fireGeoLocationEstablishedEvent();
```

Comment:
Good spot - this was when I was firing the same event in different places of the application logic (which I'm now not).